src/main/resources/templates/client-code/logging-interceptor.kt.hbs

```diff
@@ -0,0 +1,25 @@
+package {{ packages.client }}
+
+import okhttp3.Interceptor
+import okhttp3.Response
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class LoggingInterceptor : Interceptor {
+
+    private val logger: Logger = LoggerFactory.getLogger("GeneratedClient")
+
+    override fun intercept(chain: Interceptor.Chain): Response {
+        val request = chain.request()
+
+        val t1 = System.nanoTime()
+        logger.info("Client Request: ${request}")
```

Comment:
I feel like this should be a configuration option. Forcing request and response logging on the user seems like a security risk

Reply:
The logging interceptor is optional already, it's always up to the user to decide whether adding this interceptor to the okhttp client at creation.

```
                httpClient = OkHttpClient.Builder()
                        .authenticator(oauth)
                        .addInterceptor(loggingInterceptor)
                        .build()
```